controllers/storagecluster/external_ocs.go

```diff
@@ -99,7 +99,13 @@ func (r *StorageClusterReconciler) acknowledgeOnboarding(instance *ocsv1.Storage
 	}
 
 	// claims should be created only once and should not be created/updated again if user deletes/update it.
-	err = r.createDefaultStorageClassClaims(instance)
+	err = r.createDefaultStorageClassClaimsForRBD(instance)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	// claims should be created only once and should not be created/updated again if user deletes/update it.
+	err = r.createDefaultStorageClassClaimsForCephFS(instance)
 	if err != nil {
 		return reconcile.Result{}, err
 	}
```

Comment:
Let's keep all error objects local, and no need to repeat comments

```suggestion 
	// claims should be created only once and should not be created/updated again if user deletes/update it.
	if err = r.createDefaultStorageClassClaimsForRBD(instance); err != nil {
		return reconcile.Result{}, err
	}
	if err != r.createDefaultStorageClassClaimsForCephFS(instance); err != nil {
		return reconcile.Result{}, err
	}
```

Reply:
done